openwisp_monitoring/check/classes/ping.py

```diff
@@ -174,8 +174,10 @@ def _get_metric(self):
             name=check.name,
             object_id=obj_id,
             content_type=ct,
+            # TODO: Find why metric is not taking these values directly from config
             field_name='reachable',
             key=self.__class__.__name__.lower(),

```

Comment:
I don't get why this is being needed. I tried `ping` configuration across tests and there it works perfectly.